README.md

```diff
@@ -7,9 +7,97 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# assignment in opsboy file scope.
+foo = bar; # opsboy's assignment syntax. variables can be interpolated like in shell.
+
+# comment in opsboy file.
+#foo = bar; # opsboy use # as comment syntax.
+
+# rule and target.
+
+Basically,
+
+    foo {
+	directive1;
+	directive2;
+    }
+
+is a rule. And foo is the target. It's quite similar to Makefile's rules per se.
+
+We use the opsboy compiler to generate standalone Perl scripts that can build a 
```

Comment:
This line has a trailing space. Some other lines in your patch also have this problem.